### DISCUSSION TITLE
[Feature Request] Add "Local only" as a "Change post privacy" option.

### DISCUSSION THREAD

**Discussion by ghost** [mastodon/mastodon#21755] [General]
There are times when I would like a post to be seen only by the members of my server. For example, perhaps I want volunteers to serve as moderators of our instance. I would love to have "Local only" as a "Change post privacy" option.
![Change Post Privacy](https://user-images.githubusercontent.com/17678596/204160870-6a3b6b0e-e791-4584-b5ff-bbd819d1586f.png)

**Comment by afontenot** [comment]
You should post this as a feature request in https://github.com/mastodon/mastodon/issues and link it here.

**Comment by ghost** [comment]
Thanks! I have done so. Here is the link: https://github.com/mastodon/mastodon/issues/21794
